cmake/projects/tinyrefl/hunter.cmake

```diff
@@ -0,0 +1,25 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    tinyrefl
+    VERSION
+    0.4.1-p0
+    URL
+    "https://github.com/Bjoe/tinyrefl/archive/a91a16a92028c2dac7ec4ad4938d2b5b9891f94e.zip"
```

Comment:
Sorry, I need again a repo ...

Reply:
@Bjoe No problem, here you go: https://github.com/cpp-pm/tinyrefl

Reply:
Ok done.